.editorconfig

```diff
@@ -0,0 +1,85 @@
+
+# EditorConfig helps developers define and maintain consistent
```

Comment:
We should have a reference file this and a ref to it here. Is it already present in the repository template?

Reply:
Yup! https://github.com/LePetitBloc/repository-template/commit/31d17f57911dd93f5382e50ff8e22bf7f2ea1023